adding-services-to-ansible-broker.adoc

```diff
@@ -0,0 +1,71 @@
+= Adding Mobile Services to the OpenShift Ansible Broker
+
+== Prerequesites
+
+* OpenShift Origin >= 3.7
+** https://docs.openshift.org/latest/install_config/index.html
+* OpenShift Ansible Broker
+** https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-openshift-ansible-broker
+
+== Reconfigure the Broker
+
+[source,bash]
+----
+oc edit configmap broker-config -n ansible-service-broker
+----
+
+Add the following to the `registry` array:
+
+[source,yaml]
+----
+registry:
+  - type: "dockerhub"
+    name: "dh"
```

Comment:
The name must be unique, maybe it's worth pointing that out (in case users already have other registries installed)

Reply:
Good point, will add some text

Reply:
See https://github.com/aerogear/service-docs/pull/1/commits/089116753f7794d68dd78d22bdfcb31406a08520

Reply:
👍